### DISCUSSION TITLE
active?

### DISCUSSION THREAD

**Discussion by ghost** [viliusle/miniPaint#345] [Q&A] [answered]
Hi.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by viliusle** [comment, marked as answer]
Current state of this project is stable:

- Important bugs are being fixed, 
- Advanced features, that requires lots of time, postponed for future or may never, because this should not become big slow app like Photoshop.
- Some interesting and useful features will be implemented.

So I am trying to maintain this project, I do not have much time lately, but it was always like adding lots of changes, pause, add more changes, pause and so on. All contributions are welcome.

**Reply by ghost** [reply]
Hi viliusle.

> Current state of this project is stable:
> Important bugs are being fixed,
> Advanced features, that requires lots of time, postponed for future or may never, because this should not become big slow app like Photoshop.
> Some interesting and useful features will be implemented.
> So I am trying to maintain this project, I do not have much time lately, but it was always like adding lots of changes, pause, add more changes, pause and so on. All contributions are welcome.

thank you for feedback.
